src/node.cc

```diff
@@ -1581,6 +1582,10 @@ static void GetBinding(const FunctionCallbackInfo<Value>& args) {
   } else if (!strcmp(*module_v, "natives")) {
     exports = Object::New(env->isolate());
     DefineJavaScript(env, exports);
+  } else if (!strcmp(*module_v, "code_cache")) {
+    // internalBinding('code_cache')
```

Comment:
I meant to add this to the `internalBinding` but it is now in the legacy binding for debugging purposes.